src/components/shared/Backtrace.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React from 'react';
+import { filterCallNodePathByImplementation } from '../../profile-logic/transforms';
+import { getOriginAnnotationForFunc } from '../../profile-logic/profile-data';
+
+import type { Thread, IndexIntoStackTable } from '../../types/profile';
+import type { CallNodePath } from '../../types/profile-derived';
+import type { CauseBacktrace } from '../../types/markers';
+import type { ImplementationFilter } from '../../types/actions';
+
+require('./Backtrace.css');
+
+type Props = {|
+  +thread: Thread,
+  +cause: CauseBacktrace,
+  +implementationFilter: ImplementationFilter,
+|};
+
+function callNodePathForStack(
```

Comment:
Optional: This might be better collected with the other callNodePath utility functions in `src/profile-logic/`.